default.nix

```diff
@@ -111,6 +112,14 @@ let
     });
   };
 
+  # Disabling optimization for cardano-sl packages will
+  # return a build x% faster (measured in DEVOPS-1032).
```

Comment:
According to the issue both mine and Rodney's experiments outside Hydra with cache invalidation (`echo >> util/cardano-sl-util.cabal`) shows ~20% speedup for `all-cardano-sl` when optimization is disabled.

On that basis, let's `s/x%/20%/` here. https://github.com/input-output-hk/iohk-ops/pull/418 takes care of passing `fasterBuild = true;` in the Hydra jobset only for PRs.

Reply:
OK yes I have amended the commit to add that figure.